templates/master/01-master-kubelet/baremetal/files/cloudconfig.yaml

```diff
@@ -0,0 +1,6 @@
+filesystem: "root"
```

Comment:
Each file doesn't need to be added, only special platform override.

Reply:
Thanks, I didn't realize - removed